sormas-rest/src/main/java/de/symeda/sormas/rest/SormasToSormasResource.java

```diff
@@ -25,37 +25,41 @@
 
 import de.symeda.sormas.api.FacadeProvider;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasApiConstants;
-import de.symeda.sormas.api.sormastosormas.SormasToSormasCaseDto;
-import de.symeda.sormas.api.sormastosormas.SormasToSormasContactDto;
+import de.symeda.sormas.api.sormastosormas.SormasToSormasEncryptedDataDto;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasErrorResponse;
+import de.symeda.sormas.api.sormastosormas.SormasToSormasException;
+import de.symeda.sormas.api.user.UserRole;
 import de.symeda.sormas.api.utils.ValidationRuntimeException;
 
 @Path("/")
```

Comment:
Is there a good reaqson that this is not mapped to `SormasToSormasApiConstants.SORMAS_TO_SORMAS_ENDPOINT`?

Reply:
The reason is to minimize the number of constants, so I added the full path on the endpoint methods then I can use the same constant when I build the url for sending data to another sormas instance via the rest api.